src/core/Akka.Cluster/SBR/SplitBrainResolver.cs

```diff
@@ -25,23 +25,47 @@ namespace Akka.Cluster.SBR
     /// </summary>
     internal class SplitBrainResolver : SplitBrainResolverBase
     {
-        private readonly Cluster cluster;
+        private sealed class AcquireCluster : INoSerializationVerificationNeeded
+        {
+            public static readonly AcquireCluster Instance = new AcquireCluster();
+            private AcquireCluster(){}
+        }
+
+        private const string AcquireClusterKey = "acquire-cluster-key";
+        private Cluster _cluster;
 
         public SplitBrainResolver(TimeSpan stableAfter, DowningStrategy strategy)
             : base(stableAfter, strategy)
         {
-            cluster = Cluster.Get(Context.System);
```

Comment:
Can't acquire the `Cluster` in the constructor here - this risks blowing up the `ClusterCoreDaemon` who creates it and takes down the cluster node before it even starts. Need to move this into `PreStart` and then guard with messages.